src/js/main.js

```diff
@@ -116,10 +109,17 @@ _.extend(Smooch.prototype, Backbone.Events, {
 
         endpoint.sdkVersion = this.VERSION;
 
-        return this.login(options.userId, options.jwt);
+        return this.login(options.userId, options.jwt, _.pick(options, AppUser.EDITABLE_PROPERTIES));
     },
 
-    login: function(userId, jwt) {
+    login: function(userId, jwt, attributes) {
+        if (arguments.length === 2 && _.isObject(jwt)) {
```

Comment:
Yes it is for people who won't use jwt authentication.
On Nov 20, 2015 5:07 PM, "jugarrit" <notifications@github.com> wrote:

> In src/js/main.js
> <https://github.com/smooch/smooch-js/pull/135#discussion_r45525072>:
>
> >      },
> >
> > -    login: function(userId, jwt) {
> > +    login: function(userId, jwt, attributes) {
> > +        if (arguments.length === 2 && _.isObject(jwt)) {
>
> Ehhh this is rather confusing to me.
> Is it a big deal if we have the second param as 'null' when we only want
> to supply attributes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smooch/smooch-js/pull/135/files#r45525072>.
>